xonsh/__init__.py

```diff
@@ -1,5 +1,11 @@
 __version__ = '0.5.3'
 
+import sys as _sys
+if '--timings' in _sys.argv:
+    from xonsh.timings import setup_timings
+    setup_timings()
+del _sys
```

Comment:
The docs seem to indicate that it works on windows, but I didn't boot into windows to find out :) https://docs.python.org/3/library/resource.html#resource.getrusage